README.md

```diff
@@ -53,7 +54,21 @@ Configure `AzureQueueMqSettings`, there are configuration methods which can be o
 
 Requires NET core 2.x. and .NET Framework SDK 4.5 and 4.6.1
```

Comment:
Should it state 4.5.2 ? (change in csproj)

Reply:
fixed